### ISSUE TITLE
Kotlin Gradle plugin version issue for Android

### ISSUE THREAD

**Issue by SergiMS-exe** [pavel-corsaghin/react-native-leaflet#32]
Hello,

I've encountered an issue while trying to use react-native-leaflet-view in my React Native project for Android. The Android Gradle plugin in my project requires a Kotlin Gradle plugin version of 1.5.20 or higher. However, react-native-leaflet-view is dependent on org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50, which is a lower version. This mismatch is causing build failures with the following error:

`The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher. The following dependencies do not satisfy the required version: project ':react-native-leaflet-view' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50`

I've solved it going into `node_modules/react-native-leaflet-view/android/gradle.properties` and changing version. Should look like that:
`Leaflet_kotlinVersion=1.5.20`

It would be nice that you update this version to help me and other developers avoiding this issue.

Thank you!

**Comment by ssommelet21** [comment]
Hi SergiMS-exe, i did :

Leaflet_kotlinVersion=1.6.0

Because "react-native-webview" used by "react-native-leaflet" use this version.
Not sure about the need to do "gradlew clean" ...

Does  "react-native-leaflet" works ok on IOS ? thank if you can tell me.

**Comment by CarlosSTS** [comment]
Yes, it works, and we've also launched a new version. Please test it and open a new issue if you encounter any problems.
